package.json

```diff
@@ -132,11 +133,15 @@
   },
   "peerDependencies": {
     "@react-three/fiber": ">=8.0",
+    "camera-controls": ">=1.0",
     "react": ">=18.0",
     "react-dom": ">=18.0",
     "three": ">=0.137"
   },
   "peerDependenciesMeta": {
+    "camera-controls": {
+      "optional": true
+    },
```

Comment:
This should be a dependency and nothing else. Do you anticipate forking the library?

Reply:
I defined `camera-controls` as an optional peer-dep so a user that don't need camera-controls, do not get it installed when `npm i @react-three/drei` -- according to https://github.com/npm/rfcs/issues/221

Reply:
disclaimer, I'm not sure about that

Reply:
This would be tree-shaken out though. peer dependencies are for dependencies you anticipate to be installed in user-space, and that would make it no longer an implementation detail of this component.

Reply:
ok I will make `camera-controls` a simple dep

Reply:
ok done